pkg/gits/errors.go

```diff
@@ -15,6 +15,19 @@ func IsEmptyCommitError(err error) bool {
 Otherwise, please use 'git reset'`)
 }
 
+// IsEmptyCherryPickCommitError checks if the error during git cherry-pick is caused by the commit being empty at the end of the cherry-pick
+func IsEmptyCherryPickCommitError(err error) bool {
+	if err == nil {
```

Comment:
Personally, I prefer to use err != nil. But I see other functions have similar checks: `err == nil`, so I guess it's fine for now. May be in the future, it can be changed.

Reply:
Ctrl+C, Ctrl+V ;)

Reply:
Do you think ```
func IsEmptyCherryPickCommitError(err error) bool {
 	if err != nil {
 		s := err.Error()
 		return strings.Contains(s, `If you wish to commit it anyway, use:

     git commit --allow-empty

 Otherwise, please use 'git cherry-pick --skip'`)
 	}
 	return false
 }``` is more idiomatic?

Reply:
Yes, wdyt?

Reply:
Toss a coin for such a small function I reckon. But return early and often as an idiom would suggest for a function testing for certain errors, if no error text is present then a check of a nil error and early return would be more appropriate.